### DISCUSSION TITLE
F-Droid asking for permission to install from unknown sources during update

### DISCUSSION THREAD

**Discussion by gerrix1701** [bromite/bromite#1800] [Q&A] [answered]
Hi,
tried to open an issue, but it's getting closed instantly. Not sure what I'm doing wrong? Anyway, this would be the issue:

Update Bromite from version 96.0.4664.54 to 96.0.4664.104 or 96.0.4664.183 via F-Droid unattended updates does not work anymore. F-Droid is asking for permission to install software from unknown sources. Usually "F-Droid Privileged Extension" is taking care and there is no reason for F-Droid itself do update an app. This issue arises only for Bromite repository, all others work fine with "F-Droid Privileged Extension". Maybe it's something similar as with #1491 ?

Anybody else experiencing that?

**Comment by csagan5** [comment, marked as answer]
@gerrix1701 you had specified `6.0.4664.183` as a Bromite version: #1798
